src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -221,6 +227,10 @@ static void CheckForExperimentStatus(ITextView textView, Document document)
             var completionService = document.GetLanguageService<CompletionService>();
 
             var roslynTrigger = Helpers.GetRoslynTrigger(trigger, triggerLocation);
+            if (_snippetCompletionTriggeredIndirectly)
+            {
+                roslynTrigger = new CompletionTrigger(CompletionTriggerKind.Snippets);
```

Comment:
❔ Where was this condition handled prior to this change?

Reply:
Not handled in the modern completion. Missed there.

Handled in the legacy one only.

---
In reply to: [325259135](https://github.com/dotnet/roslyn/pull/38569#discussion_r325259135) [](ancestors = 325259135)

Reply:
➡️ Marking this resolved with the discussion ongoing in https://github.com/dotnet/roslyn/pull/38569#discussion_r325258740